.github/workflows/main.yml

```diff
@@ -13,15 +13,21 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out Git repository
+        uses: actions/checkout@v4
 
-      - name: Install uv
-        uses: astral-sh/setup-uv@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install dependencies
+        run: |  
+          python -m pip install --upgrade pip  
+          pip install pytest uv
+
       - name: Run tests
-        run: uv run --with python-dateutil --with six test.py
+        run: pytest tests
```

Comment:
this does seem cleaner -- I don't really get "uv run" :-)

Reply:
Failed, so put back the `uv`.